posthog/models/exported_asset.py

```diff
@@ -54,3 +55,12 @@ def filename(self):
 
     def get_analytics_metadata(self):
         return {"export_format": self.export_format, "dashboard_id": self.dashboard_id, "insight_id": self.insight_id}
+
+    @property
+    def url(self):
+        return f"{settings.SITE_URL}/exports/{self.access_token}"
```

Comment:
Problem: AFAIK SITE_URL might be unset - we don't set it by default in the chart for example.

Reply:
I think we do set it in the chart https://github.com/PostHog/charts-clickhouse/blob/d168b0f53f7540f00471dbc23c4fec8c0af2ca53/charts/posthog/templates/_helpers.tpl#L156
but its a good point that we should probably be checking more explicitly otherwise error message could be confusing

Reply:
Note we do some stuff in ingestion that affects how e.g. webhooks do it. See https://github.com/PostHog/posthog/issues/9790 and linked PRs.

Reply:
Hmmmm question is should I use that dynamic value or the configured SITE_URL? If I would use it then the naming is a little off...

I wonder what was the motivation of using a different value to `SITE_URL`?

Reply:
Added a notice about SITE_URL to the subscription modal for now. Will create a follow up issue to discuss making SITE_URL configurable